docs/module-02-01_Nonlinear-Modeling.ipynb

```diff
@@ -51,10 +51,6 @@
     "\n",
     "Nonlinear modeling simply implies that the relationship between $y$ and $x_d$ is more than a linear combination. Some common examples of nonlinear models include\n",
     "\n",
-    "**Quadratic function**: \n",
```

Comment:
the colab file accessed through the link on top of the Jupiter notebook doesn't incorporate these new changes

Reply:
Oh right.. this will also change once this PR is merged into `master`. 

See this link which opens the notebook from the `jose-edits` branch: https://colab.research.google.com/github/shawnrhoads/gu-psyc-347/blob/jose-edits/docs/module-02-01_Nonlinear-Modeling.ipynb